src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -228,12 +228,7 @@ public string NicknameFromFunctionSignatureHint(string functionSignature)
                 newName = priorNameHints[qualifiedFunction].UpgradeName;
             }
 
-            splitted = newName.Split('.');
-
-            if (splitted.Length < 2)
-                return null;
-
-            return splitted[splitted.Length - 2] + "." + splitted[splitted.Length - 1];
+            return newName;
```

Comment:
I would test zero touch loading / file loading carefully because some of our migrations tests only check there are no loaded dummy nodes(unresolved nodes) -  but they do not check that the correct number of nodes is loaded so if loading completely fails the tests still pass.